src/sign/hashedTypes.json

```
@@ -47,8 +47,8 @@
   "9cf05f3cc08b78654652fb2494d9b2beab8803b723cf1c36e1abaab7427f3823": "vote-array",
   "40bc49afc0f4084a08af0d768ff67816dde5c61550ff65008c835982a62c351e": "vote-string",
   "5339bc75243597107916a39d9235470342ad7ac8486b9424272afe4877d62f6b": "statement",
-  "8887cc25c04622fa37b81fd97615e56c6eb5f21821e3cd69ed621b0f22b6a0a7": "follow",
-  "0709a2eb5e9535594e809f7eadafe734d89ba9bf497498ede185332b6bd381d5": "unfollow",
+  "65df19852816ead53c157161c62cd1580f509f9c33d27c06d46206eb3526b8b5": "follow",
+  "150feb61a4a0f6d140f5444349c8baa068b062e7552d314a3119991e37ca6f8b": "unfollow",
```

We still need to support the old way no? Else all integrators need to pass `network`, also our UI won't work until we update the version, usually we add new hash to this list (look above)

```suggestion
  "8887cc25c04622fa37b81fd97615e56c6eb5f21821e3cd69ed621b0f22b6a0a7": "follow",
  "0709a2eb5e9535594e809f7eadafe734d89ba9bf497498ede185332b6bd381d5": "unfollow",
  "65df19852816ead53c157161c62cd1580f509f9c33d27c06d46206eb3526b8b5": "follow",
  "150feb61a4a0f6d140f5444349c8baa068b062e7552d314a3119991e37ca6f8b": "unfollow",
```

---

Seems like a good idea